librosa/core/time_frequency.py

```diff
@@ -900,6 +902,70 @@ def octs_to_hz(octs, tuning=0.0, bins_per_octave=12):
     return (float(A440) / 16)*(2.0**np.asanyarray(octs))
 
 
+def A4_to_tuning(A, bins_per_octave=12):
```

Comment:
Question for the peanut gallery: do we want this to be `A` or `A4`?  Neither is great python style, but I think the latter is more precise.  It's a positional argument, so it doesn't really matter that much, but I'm curious for people's thoughts.